steve/webedit.py

```diff
@@ -31,10 +31,12 @@
 HOST = 'localhost'
 PORT = 8000
 
+import os
+
 
 def get_data(cfg, fn):
-    data_files = load_json_files(cfg)
-    data_files = [d for d in data_files if d[0][5:] == fn]
+    data_files = load_json_files(cfg, path=True)
+    data_files = [d for d in data_files if d[0].rsplit(os.sep, 1)[-1] == fn]
```

Comment:
yes, but since we want the last segment, splitting just one from the back and taking the last is faster if there are many os.seps in d[0]